### PR TITLE
Fix warnings showing up in inference

### PR DESCRIPTION
**Pull request by lilyminium**
Fixes #

Changes made in this Pull Request:

 - Fixes the setuptools namespacing of the package, which led to `/Users/mattthompson/mambaforge/envs/nagl/lib/python3.10/site-packages/lightning_utilities/core/imports.py:13: UserWarning: Module openff was already imported from None, but /Users/mattthompson/software/openff-nagl is being added to sys.path
  import pkg_resources`

 - Fixes the pytorch tensor warning, which led to `/Users/mattthompson/software/openff-nagl/openff/nagl/molecule/_graph/_graph.py:165: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  data = {k: torch.tensor(v[edge_indices.long()]) for k, v in self.edata.items()}`

- Relates to #26 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -46,3 +46,9 @@ all-files = 1
 source-dir = docs/
 build-dir = docs/_build
 warning-is-error = 1
+
+[options]
+packages = find_namespace:
+
+[options.packages.find]
+include = ["openff.*"]
```
Comment (mattwthompson) [inline setup.cfg:54]: Presumably one or both of these sections is what fixed the `UserWarning: Module openff was already imported from None,` warning? I assumed it was simply something weird on my local machine given that I do unholy things with local installs and don't look back ...
